### ISSUE TITLE
Help regarding an issue asked 1 month ago

### ISSUE THREAD

**Issue by Pedro-Leitek** [ultralytics/yolov5#10630]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi

It has been a month since I asked for help in this forum regarding the following issue.

https://github.com/ultralytics/yolov5/issues/10353

Can someone help?

**Comment by YeopSeung04** [comment]
Hi.

I had a similar experience with you.
I also took a soccer ball data set from roboflow and learned it.
However, all similar things such as human faces, bicycle wheels, etc. were detected with soccer balls.

I found two problems.

The first question
My learning image resolution was Full HD like yours.
The resolution of the image predicting the result was 4K.

Come to think of it, it is different from full HD soccer balls and 4K soccer balls in pixels.

Yolo learns by calculating the image in pixel units in a convolutional CNN scheme.

In other words, if you learned with Full HD, it is better to use a Full HD prediction image.

The second question.
It was not enough learning.

I thought that if there was a lot of images, I would be able to learn at least
Yolo knew it was a model for performance instead of accuracy.

It would be nice to increase the number of lessons from 100.

I want you to change your way of learning.

If you learn again with --img 416 --batch 16 --epochs 100, you'll get good results.

**Comment by Pedro-Leitek** [comment]
My issue right now is not exactly the "common" false positives where the model detects similar objects as the target class. My issue right now is that, lets say I point the camera to a wall, and the whole frame appears with a bounding box saying a car was detected. But if I use a 4K imge of the same wall, the bounding box only appears at the top left of the image. I mean this is a weird behaviour. For the project this is unsustainable.

**Comment by YeopSeung04** [comment]
I understood what you said, but it is an error that occurs in the process of preventing NMS. When you use object_detection, tens of thousands draw boxes.
Also, I think it's a model's prediction error.
I also sometimes get things detected in the air. So far, artificial intelligence is the limit

**Comment by Pedro-Leitek** [comment]
Just wondering, the training images are in 640x640, and when I'm processing the image from the camera I resize it to 640x640 independently from resolution. If I train in 416x416 wouldn't the same issue happen?

**Comment by YeopSeung04** [comment]
Yes, I reduced the size of the image to 640 -> 416 and succeeded as a result of learning

**Comment by glenn-jocher** [comment]
@AI-Expert-04 thanks for sharing your experience. It's true that training on a higher resolution (e.g., 416x416) can often help improve detection accuracy. Additionally, reducing the image size to match the training resolution is indeed a common practice to ensure consistent performance during inference. If you're still experiencing issues, it might help to verify if the training data and parameters are appropriate for your use case. 🤖
